straxen/corrections_services.py

```diff
@@ -167,7 +171,7 @@ def get_pmt_gains(self, run_id, model_type, global_version):
                                  f'{type(global_version)}')
 
             # Generate an array of values and multiply by the 'global_version'
-            to_pe = np.ones(n_tpc_pmts, dtype=np.float32) * global_version
+            to_pe = np.ones(n_tpc_pmts, dtype=gain_dtype) * global_version
```

Comment:
Why can the dtype of the gains change?

Reply:
Not per se something that should be changed, therefore I made it a kwarg. Could of course also have it hardcoded in the function. I like it more than saying `np.ones(n_tpc_pmts, dtype=np.float32)` twice (also some lines down), I rather define it once.